ui/README.md

```diff
@@ -47,6 +52,7 @@ REACT_APP_AZURE_TENANT_ID=<REPLACE_WITH_YOUR_TENANT_ID>
 ### Lint
```

Comment:
Please add some contents under Lint section in README.md, so people can qucikly know what lints/code formatters have been configured. Doc should indicate where the rule are configured as well.

Reply:
I added comment about Prettier and mentioned that we're using the default rules.